### ISSUE TITLE
Unreal Gold 226 Extreme Stuttering

### ISSUE THREAD

**Issue by BlumpkinKing87** [mmdanggg2/D3D9DrvRTX#9]
The game takes forever to start up, and when it finally does only a few frames play at a time before the game freezes for several seconds (sometimes over a minute). I don't know how else to explain the issue. I followed the installation process and even made several fresh installations of the game in attempts to get this working. I've attached a couple log files from two of my most "successful" attempts, hopefully this helps.
[Unreal_d3d9.log](https://github.com/user-attachments/files/17452654/Unreal_d3d9.log)
[Unreal_d3d9BAK.log](https://github.com/user-attachments/files/17452656/Unreal_d3d9BAK.log)

**Comment by mmdanggg2** [comment]
If you temporarily remove remix (just rename the d3d9.dll) and try it, It should look like fullbright with no lighting. Does that work fine?

**Comment by BlumpkinKing87** [comment]
Yes, it launches and runs like normal and looks exactly how you described.

**Comment by mmdanggg2** [comment]
Sounds like it's set up right then but somethings wrong with remix. Make sure you've got the latest nvidia drivers etc.
You could also try in windowed mode, see if that helps. Look for `StartupFullscreen` under `[WinDrv.WindowsClient]` and set it to False

**Comment by BlumpkinKing87** [comment]
I've tried full screen and windowed. It seems to work better when windowed, but I haven't tried it in full screen since the second attempt (I've made about 10 or so attempts at this, but the entire process takes about 15 minutes before I see any visuals so it's a real pain in the ass).

**Comment by mmdanggg2** [comment]
Tracking on https://github.com/NVIDIAGameWorks/rtx-remix/issues/659
